### PR TITLE
[zh-cn] Resync configure probes page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -858,20 +858,20 @@ It will be rejected by the API server.
...
+  * [probe(s)](/docs/reference/kubernetes-api/workload-resources/pod-v1/#Probe)
 -->
 你也可以阅读以下的 API 参考资料：
 
-* [Pod](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core)
-* [Container](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core)
-* [Probe](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#probe-v1-core)
+* [Pod](/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-v1/)，尤其是：
+  * [container](/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container)
+  * [probe](/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-v1/#Probe)
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:876]: This page don't have Mandarin version.

Comment (tengqm) [reply content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:876]: Yup. Then we leave these dangling links there should be okay. We will have them soon. I believe.

**Review by howieyuen — APPROVED** [review]
/approve

**Comment by ydFu** [comment]
I open this issue https://github.com/kubernetes/website/issues/34630 for tracking.

**Pull request merged**